appmgr/src/lib.rs

```diff
@@ -89,6 +89,7 @@ pub fn server() -> Result<(), RpcError> {
     control::stop,
     logs::logs,
     properties::properties,
+    dependencies::configure,
```

Comment:
this needs another parent command to fulfill the api. as it stands it would be package.configure.dry instead of package.dependency.configure.dry